src/plugins/python/tiledbinding.py

```diff
@@ -374,6 +379,21 @@ def _decorate(obj, *args, **kwargs):
 }
 """)
 
+mod.add_function('loadTileset', 'bool', [('Tiled::SharedTileset&','sts'),('QString','file')])
+
+mod.body.writeln("""
+static bool loadTileset(Tiled::SharedTileset& sts, const QString &file)
+{
+    auto result = Tiled::TilesetManager::instance()->loadTileset(file);
+    if (!result || result->status() == Tiled::LoadingStatus::LoadingError)
```

Comment:
Could this function be implemented to simply return the `Tiled::SharedTileset`? I think for `Tiled::LoadingStatus::LoadingError` we'll want to anyway set it, since it can be useful to load the map with a tileset that failed to load entirely (allowing the user to fix up references in that tileset, for example).